packages/acs-ui-common/src/telemetry.ts

```diff
@@ -18,6 +18,16 @@ export const sanitize = (version: string): string => {
 
 /**
  * Application ID to be included in telemetry data from the UI library.
+ * Template: acXYYY/<version>
+ * Where:
+ * - X describes a platform, [r: web, i: iOS, a: Android]
+ * - YYY describes what's running on this platform (optional):
+ *    Y[0] is high-level artifact,
+ *      [0: undefined, 1: AzureCommunicationLibrary, 2: ACS SampleApp]
+ *    Y[1] is specific implementation,
+ *      [0: undefined, 1: Call Composite, 2: Chat Composite, 3: CallWithChatComposite, 4: UI Components]
+ *    Y[2] is reserved for implementation details,
+ *      [0: undefined]
```

Comment:
Please add a comment that we currently only use the format `acr/<version>` from this library.
Otherwise somebody reading this comment and then the code / tests that enforce the version might be confused at the diff.

Reply:
It already says "optional" for the YYY section, but i suppose that's pretty subtle. Will add an explicit comment, thanks.